resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/PVTestsSuite.scala

```diff
@@ -41,6 +41,7 @@ private[spark] trait PVTestsSuite { k8sSuite: KubernetesSuite =>
       kubernetesTestComponents
         .kubernetesClient
         .storage()
+        .v1()
```

Comment:
Could you elaborate a little more about this `v1`?

Reply:
Could you add some description about it too because it looks like independent from the deprecation of `DefaultKubernetesClient`.

Reply:
updated